keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/CloudDriverService.kt

```diff
@@ -38,7 +38,7 @@ interface CloudDriverService {
 
   @GET("/securityGroups/{account}/{type}/{region}/{securityGroupName}")
   suspend fun getSecurityGroup(
-    @Header("X-SPINNAKER-USER") serviceAccount: String,
+    @Header("X-SPINNAKER-USER") user: String,
```

Comment:
I changed a bunch of these parameter names based on the fact that sometimes it's not a service account, but just a user (when doing ad-hoc diff or export).